### PR TITLE
[CreateOrder] add reduce only parameter (futures tentacles must implement it)

### PR DESCRIPTION
**Pull request by techfreaque**
requires PR in trading -> [CCXT] add reduce only to create order (future exchange tentacles must implement it)
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/814

### CODE REVIEW

**Comment by techfreaque** [comment]
> ok for me after rebase 👌

done, + I added order tags

**Review by Herklos — APPROVED** [review]

File: Trading/Exchange/bybit/bybit_exchange.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ async def create_order(self, order_type: trading_enums.TraderOrderType, symbol:
                                           side=side, current_price=current_price,
                                           params=params)
 
-    async def _create_market_stop_loss_order(self, symbol, quantity, price, side, current_price, params=None) -> dict:
+    async def _create_market_stop_loss_order(self, symbol, quantity, price, side, current_price, tag: str = None, params=None) -> dict:
```
Comment (Herklos) [inline Trading/Exchange/bybit/bybit_exchange.py:158]: Why adding tag support?

Comment (GuillaumeDSM) [reply Trading/Exchange/bybit/bybit_exchange.py:158]: According to our checks, tag is not a unified ccxt argument, therefore if you want to pass it, you can do it with `order.exchange_creation_params` when necessary but it should not be in the unfirm api

Comment (techfreaque) [reply Trading/Exchange/bybit/bybit_exchange.py]: I removed it, with the coming order storage it wont be necessary for me anymore to store the tag on the exchange.

**Review by Herklos — APPROVED** [review]

**Pull request merged**